src/main/java/com/ita/if103java/ims/controller/DashboardController.java

```diff
@@ -25,8 +26,14 @@ public DashboardController(DashboardService dashboardService) {
     }
 
     @RequestMapping(value = "/warehouseLoad")
-    public List<WarehouseLoadDto> getWarehouseLoad(){
-        return dashboardService.getWarehouseLoad();
+    public List<WarehouseLoadDto> getWarehouseLoad(@RequestParam Long accountId){
```

Comment:
When Spring security will be added, please use account ID from the UserDetailsImpl class